index.html

```diff
@@ -10,147 +10,182 @@
   <link rel="stylesheet" type="text/css" href="//fonts.googleapis.com/css?family=Poppins" />
 </head>
 <body>
-  <div class="burger">
-    <div class="burger-element-1"></div>
-    <div class="burger-element-2"></div>
-    <div class="burger-element-3"></div>
+  <div id="hello-wrapper">
+    <nav>
+      <ul>
+        <li><a href="#" rel="noopener noreferrer">Hello</a></li>
+        <li><a href="#" rel="noopener noreferrer">Portfolio</a></li>
+        <li><a href="#" rel="noopener noreferrer">About</a></li>
+        <li><a href="#" rel="noopener noreferrer">Contact</a></li>
+      </ul>
+    </nav>
+    <div class="burger">
+      <div class="burger-element-1"></div>
+      <div class="burger-element-2"></div>
+      <div class="burger-element-3"></div>
+    </div>
+    <section id="hello" class="hello">
+      <h1>Charlie Giorgania</h1>
+      <p>
+        Hello! I am a software developer! I can help you build a product, feature
+        or website. Taka a look of my works. If you like what you see and have a
+        project you need coded, don’t hesitate and contact me.
+      </p>
+      <a class="collab-link" href="#" rel="noopener noreferrer">
+        <div class="collab-button">Start Collaboration</div>
+      </a>
+      <img class="deco-0" src="./Media/image_dots.svg" alt=" ">
```

Comment:
Reduce the image size as the difference shown below
#### Original design
![right](https://user-images.githubusercontent.com/58376020/125847098-5657ab7c-50b0-4936-8193-03a2fd3b4cc2.png)

#### Your Project
![wrong](https://user-images.githubusercontent.com/58376020/125847135-74c2207e-0c92-4c78-9702-120aa07f2294.png)

Reply:
Changed applied successfully.